src/frontend/packages/auth-provider/src/components/LocalLoginPage/SignupForm.js

```diff
@@ -24,7 +27,23 @@ const useStyles = makeStyles(theme => ({
   }
 }));
 
-const SignupForm = ({ redirectTo, postSignupRedirect, additionalSignupValues, delayBeforeRedirect }) => {
+/**
+ * @param postSignupRedirect
+ * @param additionalSignupValues
+ * @param delayBeforeRedirect
+ * @param {string} redirectTo
+ * @param {Object} passwordScorer Scorer to evaluate and indicate password strength.
+ *  Set to `null` or `false`, if you don't want password strength checks. Default is
+ *  passwordStrength's `defaultScorer`.
```

Comment:
It's cool that the `passwordScorer` can be set as a prop, but in reality, the only component we pass props to is `LocalLoginPage`. So please add this `passwordScorer` prop to `LocalLoginPage` and pass it down to `SignupForm` and `NewPasswordForm`.

Reply:
Oh yea, thanks for pointing out!